libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -199,8 +243,17 @@ bool AP_MotorsHeli_Single::init_outputs()
         reset_swash_servo(SRV_Channels::get_motor_function(4));
     }
 
-    // yaw servo is an angle from -4500 to 4500
-    SRV_Channels::set_angle(SRV_Channel::k_motor4, YAW_SERVO_MAX_ANGLE);
+    if (_tail_type == AP_MOTORS_HELI_SINGLE_TAILTYPE_DIRECTDRIVE_FIXEDPITCH_CW || _tail_type == AP_MOTORS_HELI_SINGLE_TAILTYPE_DIRECTDRIVE_FIXEDPITCH_CCW){
+        SRV_Channel *c = SRV_Channels::srv_channel(AP_MOTORS_MOT_4);
+        if (c != nullptr) {
+            _ddfp_pwm_min = c->get_output_min();
```

Comment:
better to grab these live from the SRV_Channel so users don't need to reboot on every change to min/max/trim

Reply:
Might as well just use set output scaled in that case.

Reply:
@IamPete1  don’t think I can do that because I need to reference the input relative to the servo trim before sending it through linearization and compensation.